modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -349,41 +349,37 @@ class Configuration(object):
             {{/servers}}
         ]
 
-    def get_host_from_settings(self, index, variables={}):
-        """Gets host URL based on the index and variables
+    def get_host_from_settings(self, index, variables=None):
+        """Get host URL based on the index and variables.
+
         :param index: array index of the host settings
-        :param variables: hash of variable and the corresponding value
+        :param variables: mapping of variable and the corresponding value
         :return: URL based on host settings
         """
-
+        variables = {} if variables is None else variables
         servers = self.get_host_settings()
 
-        # check array index out of bound
-        if index < 0 or index >= len(servers):
+        try:
+            server = servers[index]
+        except IndexError:
             raise ValueError(
-                "Invalid index {} when selecting the host settings. Must be less than {}"  # noqa: E501
+                "Invalid index {0} when selecting the host settings. Must be less than {1}"  # noqa: E501
                 .format(index, len(servers)))
 
-        server = servers[index]
         url = server['url']
 
         # go through variable and assign a value
         for variable_name in server['variables']:
-            if variable_name in variables:
-                if variables[variable_name] in server['variables'][
-                        variable_name]['enum_values']:
-                    url = url.replace("{" + variable_name + "}",
-                                      variables[variable_name])
-                else:
-                    raise ValueError(
-                        "The variable `{}` in the host URL has invalid value {}. Must be {}."  # noqa: E501
-                        .format(
-                            variable_name, variables[variable_name],
-                            server['variables'][variable_name]['enum_values']))
-            else:
-                # use default value
-                url = url.replace(
-                    "{" + variable_name + "}",
-                    server['variables'][variable_name]['default_value'])
+            server_variable = server['variables'][variable_name]
+            value = variables.get(variable_name, server_variable['default_value'])
+
+            if 'enum_values' in server_variable and value not in server_variable['enum_values']:
```

Comment:
The line lengths are throwing linting errors our continuous integration tests per:
```
petstore_api/configuration.py:321:80: E501 line too long (82 > 79 characters)
            value = variables.get(variable_name, server_variable['default_value'])
                                                                               ^
petstore_api/configuration.py:323:80: E501 line too long (96 > 79 characters)
            if 'enum_values' in server_variable and value not in server_variable['enum_values']:
```
Can you make sure your long lines get shortened to 79 characters or less?
Remember that if we are doing assignment we can use parens like:
```
blah = (
   "some very long value here.........."
)
```